doc/SONiC_static_route_hdl.md

```diff
@@ -81,28 +81,46 @@ nexthop             = IPv4Address / IPv6Address; List of gateway addresses;
 ifname              = List of interfaces
 distance            = {0..255};List of distances.
                       Its a Metric used to specify preference of next-hop
-                      if this distance is not set 0 will be set to maintain the set;
-nexthop-vrf         =  1\*15VCHAR ; list of next-hop VRFs. It is set only if interface is not
+                      if this distance is not set, 0 will be set to maintain the set;
+nexthop-vrf         =  1\*15VCHAR ; list of next-hop VRFs. It should be set only if ifname or nexthop IP  is not

```

Comment:
We should not allow user to configure "list of next-hop VRFs" since ECMP or multi-metric routes across VRF is not supported. Even FRR & Kernel doesn't allow ECMP across next-hop VRFs. 
If user tries to configure static route with different next-hop VRF, FRR updates existing static route with new next-hop VRF.

Reply:
FRR allowes it . Please see below for prefix 40.0.0.0/8. 

sonic(config)# do show ip route
Codes: K - kernel route, C - connected, S - static, R - RIP,
       O - OSPF, I - IS-IS, B - BGP, E - EIGRP, N - NHRP,
       T - Table, v - VNC, V - VNC-Direct, A - Babel, D - SHARP,
       F - PBR, f - OpenFabric,
       > - selected route, * - FIB route, q - queued route, r - rejected route, # - not installed in hardware

C>*  10.0.0.0/8 is directly connected, eth0, 20:47:32
S    40.0.0.0/8 [200/0] via 2.2.2.2, Ethernet0(vrf Vrf-RED), 00:00:02
                        via 1.1.1.2, Ethernet80(vrf Vrf-GREEN), 00:00:02
S>*  40.0.0.0/8 [1/0] is directly connected, eth0, 00:03:49

Reply:
Sure, then I think we should allow configuration of ECMP static leak routes across VRFs.